### DISCUSSION TITLE
I got some questions

### DISCUSSION THREAD

**Discussion by RaptaG** [dani-garcia/vaultwarden#3668] [Q&A] [answered]
1. Is there a Linux and Android client? If yes where?
2. Can I port my account from https://bitwarden.com to a VaultWarden server and how?

**Comment by stefan0xC** [comment, marked as answer]
Check out https://bitwarden.com/download/ and you can [export](https://bitwarden.com/help/export-your-data/) and [import](https://bitwarden.com/help/import-data/) your vaults.
